scripts/ci.sh

```diff
@@ -15,8 +15,8 @@ if grep -q 'sdk: flutter' "./pubspec.yaml"; then
       flutter format --set-exit-if-changed test
       flutter analyze --no-current-package test
       flutter test --no-pub --coverage
+      cp ./coverage/lcov.info ../../$package.lcov
```

Comment:
This would break the regular CI, as it will not go to the if.

Reply:
My understanding is that `./coverage/lcov.info` is generated by `flutter test --no-pub --coverage`. So it should work?

Reply:
Oh, I see.